ext/fast_osc/fast_osc_wrapper.c

```diff
@@ -68,6 +73,45 @@ uint64_t ruby_time_to_osc_timetag(VALUE rubytime) {
   return timetag;
 }
 
+VALUE osc_timetag_to_ruby_time(uint64_t timetag) {
+  uint32_t secs = timetag >> 32;
+  uint32_t frac = timetag & 32;
+
+  // Time.at(seconds, microsecs_with_frac)
+  VALUE c_time = rb_const_get(rb_cObject, rb_intern("Time"));
+  VALUE rb_time = rb_funcall(c_time, rb_intern("at"), 2, UINT2NUM(secs - JAN_1970), DBL2NUM(1.0 * frac / pow(2, 32)));
```

Comment:
in a previous commit you replaced `pow(2, 32)` with constant. Is there a reason you reverted to using `pow` again?

Reply:
This is an old commit, it's ok in the latest version in the PR